### ISSUE TITLE
[Bug]: runtimejobs does not detect long running jobs

### ISSUE THREAD

**Issue by mowgli** [NETWAYS/check_bareos#40]
### Please try to fill out as much of the information below as you can. Thank you!

- [X] Yes, I've searched similar issues on GitHub and didn't find any.

### Which version contains the bug?

2.1.0

### Describe the bug

```
> /usr/local/lib/nagios/plugins/check_bareos -U bareos --password-file /etc/bareos/db-password job --runTimeJobs -t 2 -w 1 -c 3
[OK] - 0.0 Jobs are running longer than 2 days|bareos.job.count=0.0;1;3;;
```

However, I currently have one job that is running for more than 3 days. (Started 20.9. 9:41)

**Comment by martialblog** [comment]
Hi, 

just to clarify, the default jobs that are checked are `'C': 'Job created but not yet running'`. Maybe the jobs are in a different state? You can set the state with the `-st` parameter

Maybe we should add the states to the output to avoid confusion.

**Comment by mowgli** [comment]
Yes, that would be helpful. "running" means, well, running for me and not waiting.

**Comment by martialblog** [comment]
I'll try to improve the output.

But I think you also came across a bug. Just had a look at the code and there could be an issue with the SQL query.

Thanks for reporting the issue

**Comment by martialblog** [comment]
Ok upon closer look the query looks alright and seems to work with Bareos 23.0.4.

I extended the output of the plugin so that it is clearer what is happening. As mentioned in the issue, I think the the -st parameter is what you need.

**Comment by martialblog** [comment]
PR https://github.com/NETWAYS/check_bareos/pull/41 has improved the output and should make things clearer in the future. 

Waiting for user feedback. Otherwise we close this issue

**Comment by martialblog** [comment]
Is included in the coming release
